### ISSUE TITLE
Introduce a nicknames cache

### ISSUE THREAD

**Issue by swentel** [swentel/indigenous-android#134]
see https://indieweb.org/nicknames-cache

I should open up a micropub extension request.

Related with https://github.com/swentel/indieweb/issues/245

**Comment by EdwardHinkle** [comment]
Hey, what all are you thinking about doing here? I’m thinking about this for iOS as well. One main thing I’m thinking about is adding items in to the nicknames cache by essentially sending a Micropub request to my server with an h-card object.

As far as using the nicknames cache, I’d like to use it to be able to autofill @-mentions or tags.

**Comment by swentel** [comment]
> I’m thinking about is adding items in to the nicknames cache by essentially sending a Micropub request to my server with an h-card object.

I saw you talking with aaron a few days ago (which is essentially I learned about this heh). And I forgot about sending. What I was thinking initially was the other way around, as in: sending a request to the micropub endpoint, something like ?q=nicknames, so they can be used for autofill in the app.

However, sending a h-card to the endpoint sounds interesting as well - I have to check the storage on the drupal side, and check the spec, but that's related with https://github.com/swentel/indieweb/issues/245
